### PR TITLE
Adds .ps1 script to execute LST_Bench

### PR DESCRIPTION
**Pull request by netonjm**
Adds support to launch LST_Bench in windows using ps1 script

Fixes https://github.com/microsoft/lst-bench/issues/33

### CODE REVIEW

**Comment by jcamachor** [comment]
Thanks for your contribution @netonjm ! Would you mind to update the information in `README.md` to include how to run on Windows? We should also enable Windows CI, if you could create a GH issue, that would be great too.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,45 @@ LST-Bench is a framework that allows users to run benchmarks specifically design
...
+
+Confirm with mvn -v in a new shell. The result should look similar to:
+
+```
+Apache Maven 3.9.2 (c9616018c7a021c1c39be70fb2843d6f5f9b8a1c)
+Maven home: /opt/apache-maven-3.9.2
+Java version: 1.8.0_45, vendor: Oracle Corporation
+Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre
+Default locale: en_US, platform encoding: UTF-8
+OS name: "mac os x", version: "10.8.5", arch: "x86_64", family: "mac"
```
[the remark this answers is not in the document]
Comment (netonjm) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -41,7 +77,8 @@ mvn package -Pspark-oss
 ```
 
 ### How to Run
-After building LST-Bench, run `launcher.sh` to display the usage options.
+
+After building LST-Bench, if you are on linux/mac run `launcher.sh` or open a Powershell `launcher.ps1` if you are on Windows to display the usage options.
```
Comment (anjagruenheid) [inline README.md:81]: Nit: linux -> Linux, mac -> Mac :)

**Review by anjagruenheid — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (netonjm) [reply README.md]: fixed

**Pull request merged**